go/sys/fs/fs.go

```
@@ -168,12 +168,12 @@ func FileExists(ctx context.Context, path string) bool {
 // Remove removes a file using sudo rm -f.
 // This is a best-effort operation that doesn't return errors.
 func Remove(ctx context.Context, path string) {
-	exec.Sudo(ctx, "rm", "-f", path)
+	exec.Privileged(ctx, "rm", "-f", path)
```

golangci-lint (2.11.4)

[error] Error return value of `exec.Privileged` is not checked

(errcheck)